hastexo/public/js/main.js

```diff
@@ -444,6 +444,15 @@ function HastexoXBlock(runtime, element, configuration) {
                     dialog.find('input.ok').one('click', function() {
                         $.dialog.close();
                     });
+                    var hints_title = dialog.find('.hints_title').hide();
+                    var hints = dialog.find('.hints').empty().hide();
+                    if (data.errors.length > 0) {
+                        $.each(data.errors, function(i, error) {
+                            hints.append($('<li>', {text: error}));
```

Comment:
You should probably consider multi-line output on stderr here. I suppose you can either split the `error` content at newlines here and replace those with `<br/>`, _or_ (to be continued below)

Reply:
This is the better place to do it, I believe.  It's a display concern, so better to handle it in the code that does the displaying.

Reply:
Yep, I don't have a preference either way. Fine with either.